src/components/Navbar/SettingsDrawer/TranslationSection.tsx

```diff
@@ -14,10 +14,10 @@ import {
   selectQuranReaderStyles,
 } from 'src/redux/slices/QuranReader/styles';
 import {
-  selectTranslations,
+  selectSelectedTranslations,
   setSelectedTranslations,
 } from 'src/redux/slices/QuranReader/translations';
-import { numbersToStringsArray, stringsToNumbersArray } from 'src/utils/array';
+import { areArraysEquals, numbersToStringsArray, stringsToNumbersArray } from 'src/utils/array';
```

Comment:
it's from before my bad but `areArraysEquals` should be renamed to `areArraysEqual`

Reply:
thanks for catching this. Fixed